sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkSpanBuilder.java

```diff
@@ -83,12 +83,24 @@ public SpanBuilder setSpanKind(SpanKind spanKind) {
 
   @Override
   public SpanBuilder addLink(SpanContext spanContext) {
+    if (spanContext == null || !spanContext.isValid()) {
+      return this;
+    }
     addLink(LinkData.create(spanContext));
     return this;
   }
 
   @Override
   public SpanBuilder addLink(SpanContext spanContext, Attributes attributes) {
```

Comment:
Should we also document in the API, same as you din in the specs:

"Implementation MAY ignore ...."

Reply:
Sure